spec/controllers/admin/cms/collections_controller_spec.rb

```diff
@@ -1,5 +1,163 @@
 require 'rails_helper'
 
-RSpec.describe Admin::CollectionsController, type: :controller do
+RSpec.describe Admin::Cms::CollectionsController, type: :controller do
+  let(:vendor) { Vendor.create!(uid: 'voice_base', name: 'VoiceBase') }
```

Comment:
booo for using real model instances instead of doubles in controller specs

(although that's just a personal preference/best practice thing)